modules/oe_editorial_corporate_workflow/modules/oe_editorial_corporate_workflow_translation/oe_editorial_corporate_workflow_translation.info.yml

```diff
@@ -2,7 +2,7 @@ name: OpenEuropa Editorial Corporate Workflow Translation
 description: Integrates the editorial workflow with OE Translation
```

Comment:
Moule can be remved, also the corporate_workflow_translation_poetry.info.yml file above